### ISSUE TITLE
[20220403] Weekly AI ArXiv 만담 - 46회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#46]
- News
  - Deadline
    - Interspeech 2022, 수고 많으셨습니다!
    - ICML 22: Review out (4. 7, 저녁)
  - [인공지능과 지식재산백서](https://www.kipo.go.kr/ko/kpoBultnDetail.do?ntatcSeq=16558&aprchId=BUT0000048&searchCondition=&pageIndex=1&sysCd=SCD02&menuCd=SCD0200640&keyword=)
    - 관련뉴스: ['인간이 만든' 인공지능, 이제는 발명가 꿈꾼다](https://news.naver.com/main/ranking/read.naver?oid=031&aid=0000662061)
  - [Meta의 AI 인력 유출](https://www.cnbc.com/2022/04/01/metas-ai-lab-loses-some-key-people.html)
- Arxiv
  - [Training Compute-Optimal Large Language Models](https://arxiv.org/abs/2203.15556)
    - Hyperscale LM (LLM) 에서 모델 크기와 학습 데이터량의 상관관계를 어마무시한 실험으로 분석 (DeepMind님 감사합니다)
    - 가정: 모델 크기가 2배 늘면 학습데이터도 두배 늘어야 한다. --> Chinchilla 제안 (작고 엄청 많이 먹는 뚱뚱한 쥐)
    - 이론적인 최적의 FLOP수 계산. Kaplan et al. (2021) 의 scaling law보다 효율적 가능
    - 70B Chinchilla 로 280B Gopher, 175B GPT3, 530B Turning-NLG 보다 더 강력한 성능
    - 그런데 파라미터를 줄일 때 layer수는 고정하고 head와 hidden dimension 수를 줄임 --> 역시 depth가 중요함
![image](https://user-images.githubusercontent.com/11782739/161364154-fb6c751e-21be-4ef7-bbf3-d1749de2ba74.png)
![image](https://user-images.githubusercontent.com/11782739/161364237-3fdcbaef-0fe0-4c29-9048-926fc73c6f88.png)
![image](https://user-images.githubusercontent.com/11782739/161364247-329b8e95-a393-4224-bb47-1e013ba93423.png)

  - [Three things everyone should know about Vision Transformers](https://arxiv.org/abs/2203.09795)
    - 제목 그대로 ViT 학습할 때 알아두면 유용할 3가지 (from Meta AI Paris + Inria)
    - 1) Transformer Block을 parallel하게 해도 성능이 유지된다.
    - 2) ImageNet-1k 나 작은 데이터 finetuning 시에는 MHSA 부분만 update해도 경쟁력 있다.
    - 3) MLP기반의(여기선 hierarchical MLP구조) 패치 만들면 BERT스타일 Masked reconstruction 학습시 효과적이더라
    - 추가: 파라미터, 속도, 메모리 대부분 depth에 비례, width에 제곱에 비례한다.
![image](https://user-images.githubusercontent.com/11782739/161382548-8c5c336f-8b26-4820-8880-b02a5c754efc.png)
![image](https://user-images.githubusercontent.com/11782739/161382585-b780b269-01d1-4ce0-bf4b-c4f8df9f78af.png)
![image](https://user-images.githubusercontent.com/11782739/161382615-6f0e48f3-9b76-456c-904d-aee234d7b16a.png)
    
  - [Visual Prompting: Modifying Pixel Space to Adapt Pre-trained Models](http://arxiv.org/abs/2203.17274)
    - LLM만 prompte-based learning 가능한 것이 아니다! 이미지도 가능! (from MIT, 1저자 방효진님은 NAVER AI Lab 인턴 출신)
    - 간단한 pixel patch 를 prompt처럼 패딩시키고 새로운 태스크에 적용하면 모델 추가 학습없이 정확도 향상
    - LLM에서 prompt는 자연어라 다루기 쉬운데 비해 image prompt는 직관적/해석 어려움이 있는 데 이걸 풀수 있다면..
    - In distribution 보다 out-of-distribution 성능이 확실히 좋고 CLIP에선 Text prompt와도 함께 사용 가능
    - https://hjbahng.github.io/visual_prompting/
![image](https://user-images.githubusercontent.com/11782739/161383052-d0a7be3d-3155-446a-be80-94615acd228c.png)
![image](https://user-images.githubusercontent.com/11782739/161383066-fefe6502-7a6f-47db-83a2-b19b1329cece.png)
![image](https://user-images.githubusercontent.com/11782739/161383114-30644a90-d7c8-4bb6-8040-b15f3c039ad5.png)

- 흥미있는 연구
  - [Bringing Old Films Back to Life](http://arxiv.org/abs/2203.17276)
    - MS에서 나온 흑배영화를 칼라로 만들어주는 ! (CVPR 2022)
    - https://github.com/raywzy/Bringing-Old-Films-Back-to-Life
  - [FindIt: Generalized Localization with Natural Language Queries](https://arxiv.org/abs/2203.17273)
    - Visual grouding, text-based localization, object detection 등 다양한 이미지내 semantic localization을 한큐에 하는 구글 브레인 연구 (from Google Brain)
   - [The Role of ImageNet Classes in Fréchet Inception Distance](https://arxiv.org/abs/2203.06026)
     - FID 계산할 때 ImageNet-pretrained InceptionNet 활용에 대한 고찰 (from NVidia)
   - [A Roadmap for Big Model](https://arxiv.org/abs/2203.14101)
     - 중국 BAAI에서 나온 Hyperscale AI에 대한 고찰 논문. 요즘 이런거 쓰는 것이 유행인 것인가...

**Comment by ghlee0304** [comment]
- News 
    -  [Generating chit-chat including laughs, yawns, 'ums,' & other nonverbal cues from raw audio](https://ai.facebook.com/blog/generating-chit-chat-including-laughs-yawns-ums-and-other-nonverbal-cues-from-raw-audio)
- Arxiv 
    - [Generative Spoken Dialogue Language Modeling](https://arxiv.org/abs/2203.16502) 
    - [DRSpeech: Degradation-Robust Text-to-Speech Synthesis with Frame-Level and Utterance-Level Acoustic Representation Learning](https://arxiv.org/pdf/2203.15683.pdf)

**Comment by nick-jhlee** [comment]
- News
  - Kahn-Kalai Conjecture (2006) solved!!
    - [A Proof of the Kahn-Kalai Conjecture](https://arxiv.org/abs/2203.17207) (Jinyoung Park, Huy Tuan Pham)
    - [Thresholds and expectation thresholds](https://arxiv.org/pdf/math/0603218.pdf) (Jeff Kahn, Gil Kalai)
  - Transactions on Machine Learning Research (TMLR) now open to submissions!
    - https://jmlr.org/tmlr/news/2022/launch.html
  - AISTATS 2022 held online! (04/28~04/30)
   
<img width="400" alt="Screen Shot 2022-03-29 at 1 07 41 AM" src="https://user-images.githubusercontent.com/38721329/161415774-4a5f2b28-14b4-4b78-b304-905b24498e2c.png"> <img width="400" alt="Screen Shot 2022-03-29 at 1 09 02 AM" src="https://user-images.githubusercontent.com/38721329/161415776-95400ffb-c711-437d-a6ad-fb4f1d7fe573.png">
<img width="400" alt="Screen Shot 2022-03-29 at 1 09 32 AM" src="https://user-images.githubusercontent.com/38721329/161415778-189ca08d-3f9f-4a3b-a2cc-36c18481468b.png"> <img width="400" alt="Screen Shot 2022-03-29 at 1 06 43 AM" src="https://user-images.githubusercontent.com/38721329/161415772-b11b065c-391e-43e4-9ae3-d21e30aa30fd.png">
 
 
 - Papers
   - [Multi-armed Bandit Algorithm against Strategic Replication]() (AISTATS 2022)
     - LINE Plus Corporation, POSTECH
   - [Non-separable Spatio-temporal Graph Kernels via SPDEs]()
   - Nonstochastic Bandits and Experts with Arm-Dependent Delays
   - Differentiable Bayesian inference of SDE parameters using a pathwise series expansion of Brownian motion
   
- Extra Papers (발표용은 아니지만, 여쭤보시면 약간의 summary는 해드리겠습니당 ㅎㅎㅎ)
  - [Fast and accurate optimization on the orthogonal manifold without retraction](https://arxiv.org/pdf/2102.07432.pdf)
    - CNRS, ENS PSL
 - [Neural Contextual Bandits without Regret](https://arxiv.org/pdf/2107.03144.pdf)
   - ETH Zurich
 - [Norm-Agnostic Linear Bandits](https://www.math.arizona.edu/~sethuram/papers/2022gss.pdf)
   - University of Arizona
 - [Joint Efficient and Optimal Algorithms for Logistic Bandits](https://arxiv.org/pdf/2201.01985.pdf)
   - Criteo AI Lab, University of Arizona
 - [Maillard Sampling: Boltzmann Exploration Done Optimally](https://arxiv.org/pdf/2111.03290.pdf)
   - University of Arizona
 - [Efficient Kernel UCB for Contextual Bandits](https://arxiv.org/pdf/2202.05638.pdf)
   - Criteo AI Lab, NYU Center for Data Science, INRIA
 - [Minimal Expected Regret in Linear Quadratic Control](https://arxiv.org/pdf/2109.14429.pdf)
   - KTH

**Comment by hollobit** [comment]
### When people tell me "AI is moving so fast", I think of this image
https://twitter.com/fchollet/status/1507199130665332816
<img  src="https://user-images.githubusercontent.com/998803/161426033-16520c4c-70c8-4b57-962f-6d7afeadaacd.png">

**Comment by terryum** [comment]
- News
    - AI 클라우드 플랫폼 시장 격화
        - [NHN 클라우드 출범](http://www.aitimes.kr/news/articleView.html?idxno=24700)
        - [KT클라우드 출범](http://www.aitimes.kr/news/articleView.html?idxno=24695)
        - [KB금융, AI클라우드로 'KB미래컨택센터' 구축](http://www.aitimes.kr/news/articleView.html?idxno=24652)
        - [중국은행, AI로 변화의 바람](http://kr.news.cn/2022-03/28/c_1310532004.htm)
    - [AI 상장사 13곳 분석 : 매출 확대](http://www.aitimes.com/news/articleView.html?idxno=143662&page=3&total=9292)
  
![image](https://user-images.githubusercontent.com/12528769/161426882-8e1afd24-60ef-408a-8ab3-5892c6101f40.png)

**Comment by veritas9872** [comment]
**SharpContour: A Contour-based Boundary Refinement Approach for Efficient and Accurate Instance Segmentation**

CVPR 2022에 Accept된 논문을 공유해드립니다. 기존의 instance segmentation은 가장자리 부근에서 performance가 contour를 잘 따르지 않는 문제점이 있었는데 본 논문에서는 Instance-aware Point Classifier (IPC)를 segmentation network의 feature로부터 학습해 iterative boundary update process를 통해서 경계를 다듬습니다.

특히 medical imaging 등 fine detail segmentation이 중요한 task에서 많이 유용할 것으로 생각됩니다. 코드도 조만간 오픈소스될 예정이라고 합니다.

Website: https://xyzhang17.github.io/SharpContour

Arxiv: https://arxiv.org/abs/2203.13312

![image](https://user-images.githubusercontent.com/33523965/161428029-2c6c35e5-66b8-4eeb-acd6-43ff10d5d681.png)

![image](https://user-images.githubusercontent.com/33523965/161428162-21665b02-3361-44c3-9dd4-6ceaaaa8c730.png)

![image](https://user-images.githubusercontent.com/33523965/161428181-1ba86d40-022c-47c3-92b1-8cf4472e5e2d.png)

**Comment by hollobit** [comment]
수십억 달러 언어 AI 스타트업의 물결이 오고 있다
https://www.forbes.com/sites/robtoews/2022/03/27/a-wave-of-billion-dollar-language-ai-startups-is-coming/?sh=653eb4472b14
>- 첫번째 범주는 핵심 범용 NLP 기술을 개발하고 제공하는 플레이어 : OpenAI, Cohere, Hugging Face, AI21 Labs, Primer. Inflection AI
>- 두번째 범주는 검색 : You.com, ZIR AI, Algolia, Constructor.io, Twelve Labs
>- 세번째 범주는 글쓰기 도우미 : Grammarly, Textio, LitLingo, Writer, CopyAI
>- 네번째 범주는 언어 번역 : 블랑, KUDO, Lilt, NeuralSpace
>- 다섯번째 범주는 영업 인텔리전스 : Gong, Chorus.ai
>- 여섯번째 범주는 챗봇 도구 및 인프라 : Ada Support, Rasa, Thankful
>- 대화형 음성 도우미 : Duplex, Replicant, AI Rudder
